torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -277,14 +277,6 @@ FusionKernelRuntime::FusionKernelRuntime(
   } else {
     auto complete_fusion_heuristic = maybe_complete_fusion_heuristic.value();
 
-    // Translate welfords if apply
```

Comment:
If I'm understanding correctly, if a welford makes its way into complete fusion it will be a reduction complete fusion since persistent now doesn't allow it. 

Do we still want the case where the complete fusion is welford reduction but we translate it to persistent?

Reply:
That still can happen, we just run it through segmentation for the translation. I thought it was better to unify this since somehow welford was making its way into a persistent kernel. This seems to successfully prevent any chance of that happening.